client/utils/account-fees.tsx

```diff
@@ -222,19 +222,19 @@ export const formatAccountFeesDescription = (
 		...customFormats,
 	};
 
+	// Some payment methods doesn't have base percentage rate. In this case, the lowest rate will be shown as a start value
+	let displayFeePercentageRate = baseFee.percentage_rate;
+	if ( displayFeePercentageRate <= 0 ) {
+		displayFeePercentageRate =
+			additionalFee.percentage_rate < fxFee.percentage_rate
+				? additionalFee.percentage_rate
+				: fxFee.percentage_rate;
+	}
 	const feeDescription = sprintf(
 		formats.fee,
-		formatFee(
-			baseFee.percentage_rate +
-				additionalFee.percentage_rate +
-				fxFee.percentage_rate
-		),
-		formatCurrency(
-			baseFee.fixed_rate + additionalFee.fixed_rate + fxFee.fixed_rate,
-			baseFee.currency
-		)
+		formatFee( displayFeePercentageRate ),
+		formatCurrency( baseFee.fixed_rate, baseFee.currency )
```

Comment:
Is sending only `fixed_rate` here an intended change? I am not sure about this change tbh and would appreciate another pair of eyes on this. cc @jessy-p who might have more context.

Reply:
To explain simply, the change is that, instead of showing fee as "baseFee.percentage_rate + additionalFee.percentage_rate + fxFee.percentage_rate" we will show "From baseFee.percentage_rate", so number doesn't look too high. By using the keyword "From" we are saying, it could be more too.

Reply:
Similarly for fixed_rate, we use the most common and usually applied baseFee.fixed_rate instead of adding all the rates and showing that value